lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -6111,6 +6111,7 @@ namespace Js
         Assert(scriptContext->GetConfig()->IsES6PromiseEnabled());
 
         FunctionInfo* functionInfo = &Js::JavascriptPromise::EntryInfo::CapabilitiesExecutorFunction;
+        functionInfo->SetAttributes((FunctionInfo::Attributes)(functionInfo->GetAttributes() | FunctionInfo::Attributes::ErrorOnNew));
```

Comment:
doesn't this already have ErrorOnNew attribute?

Reply:
Yes it does - I'll remove it. Also for few others.